plugins/generator-1.19.4/forge-1.19.4/templates/elementinits/items.java.ftl

```diff
@@ -40,7 +40,7 @@ package ${package}.init;
 <#assign hasBlocks = false>
 <#assign hasDoubleBlocks = false>
 
-<#if w.hasItemsWithCustomProperties()>
+<#if w.getGElementsOfType("item")?filter(e -> e.customProperties?has_content)?size != 0 || w.getGElementsOfType("tool")?filter(e -> e.toolType == "Shield")?size != 0>

```

Comment:
It would make sense to assign the result of this to an FTL variable. This would reduce iterations from 4x to 2x

Reply:
:heavy_check_mark: Fixed (is the name fine?)

Reply:
Yep